README.md

```diff
@@ -95,6 +95,22 @@ Here is an examples of toplevel OCaml code:
     42
     ```
 
+### File sync
+`mdx` is also capable of two-way synchronization of fenced code blocks with
```

Comment:
I wouldn't mention that it can be done from the md file to the ml one as we're planning on dropping that.

Reply:
Done in 54fcc23da04fa1542910cf0caec3d5688123fe8e